### ISSUE TITLE
Klondike scoring system changed? 

### ISSUE THREAD

**Issue by alekksander** [TobiasBielefeld/Simple-Solitaire#87]
Since may, I cannot get anywhere in the score board. Is the pointing system changed?

**Comment by alekksander** [comment]
![screenshot_2017-11-06-19-08-56](https://user-images.githubusercontent.com/3148302/32456673-b322c88e-c326-11e7-8bf6-f491039d63f0.png)

**Comment by TobiasBielefeld** [comment]
Yes, i had to in version 3.8.5 (see issue #66) The bonus scoring wasn't correct and way to much (nearly always more than the actual score) So I corrected that. 
There was also no way to adjust the saved high scores, so the only solution is to delete the old scores :c

**Comment by alekksander** [comment]
Okay, thank You for explaining.

**Comment by alekksander** [comment]
by the way, is it intended that the clock is starting even if not touching any cards? 
is it possible to revert counting game as lost if i don't touch any card, and ask for new deck?

the way described by me, was how it worked back before one of the updates.

**Comment by TobiasBielefeld** [comment]
well I thought: if you can see the cards,  you can already plan your moves, so the clock should start. But if you think it would be better to start the clock only after moving the first card, i can change that.

But as long as you don't move cards, the game counter won't increase!

**Comment by alekksander** [comment]
> you can already plan your moves, so the clock should start.

You're correct. it is more fair now.

the second question was my mistake. maybe in previous versions it counted as lost, or i confused some numbers ;)

best regards!

**Comment by TobiasBielefeld** [comment]
ok, glad I could help :)

**Comment by alekksander** [comment]
Dont want to create new issue, as this is related. Game started, i messed up, need to start over. Is it on purpose, that it count game as lost, even if its the same deal?

**Comment by TobiasBielefeld** [comment]
It is, but if you think it should be changed, I'll change that :)
